System/Win32/Types.hs

```diff
@@ -26,10 +26,10 @@ import Data.Char (isSpace)
 import Data.Int (Int32, Int64)
 import Data.Maybe (fromMaybe)
 import Data.Word (Word8, Word16, Word32, Word64)
-import Foreign.C.Error (getErrno, errnoToIOError)
+import Foreign.C.Error (Errno(..), errnoToIOError)
```

Comment:
Just `Errno` should be enough we don't need the actual constructors (so no `(..)`).